dataBaseLogic.py

```diff
@@ -1,7 +1,7 @@
 from flask import Flask, g
 import sqlite3
 
-DATABASE = 'C:\\Users\\alexa\\PycharmProjects\\swen444\\donatoro.sqlite'
+DATABASE = 'C:\\Users\\Joshua\\Desktop\\swen444\\donatoro.sqlite'
```

Comment:
Is this acceptable? We should really be pulling this from some sort of configuration file.